pkgs/by-name/wo/workrave/package.nix

```diff
@@ -86,7 +82,7 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   meta = with lib; {
-    broken = (stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64);
+    broken = stdenv.hostPlatform.isLinux && stdenv.hostPlatform.isAarch64;
```

Comment:
It builds fine on `aarch64-linux` now, so this can just be dropped (in a separate commit, please).